README.md

```diff
@@ -1,124 +1,70 @@
-
-</div>
 
-#### Data flow
+> **_[Information]_**  Other actions which are typical for the discipline of machine learning shall deliberately be excluded, e.g. training and data exploration.
 
-When a request comes in (1) via the http-server (http) it is routed (2) to the actor (M). The actor (M) registers (3) at the inference engine (IE) in case it did not do so before and provides the data (3) to the inference engine (IE). The result is passed (4) from the inference engine (IE) back to the actor (M) which, in turn, passes (5) the result to the http-server (https) where it is sent as a response.
+## Design decisions
 
-<div style="width: 40%; height: 25%">
+### Inference API actor
 
-![legend for most basic design](images/most_basic_design_data_flow.png "Most basic design of an example application doing inference")
+The Inference API actor (IAPI) shall
 
-</div>
+1. be linked to a HTTP server (http)
+2. expose a public API for requesting inference actions.
+3. validate its input
+4. make actor-to-actor calls to an inference actor (I)
+5. translate internal results to public HTTP output
 
-### More advanced design
+### Inference Actor
 
-A more advanced design additionally introduces an inference client (IC). The inference client (IC) communicates with the http-server (http) and the inference engine (IE) and may be regarded as a *reverse proxy* in that it deals with the opaque handles of the inference engine (IE) alias [GraphExecutionContext](https://github.com/Finfalter/wasmCloudArtefacts/blob/main/interfaces/mlinference/rust/src/mlinference.rs) internally but deals with (nick) names of AI models externally. 
+The Inference Actor (I) shall
 
-> **_[Note]_**  The use of an inference client (IC) allows an easy deployment of multiple actors (M). However, the interface of such a design would differ from an interface of the basic design.
+6. await actor-to-actor calls to perform actions
+7. perform any necessary validation on an incoming call
+8. request inference processing from the inference engine (IE) and return results
 
-<div style="width: 40%; height: 25%">
+### Inference Engine Capability Provider
 
-![most basic design](images/more_advanced_design.png "Most basic design of an example application doing inference")
-</div>
+The Inference Engine Capability Provider (IE) shall
 
-<div style="width: 40%; height: 25%">
+9. expose a [WASI-NN](https://github.com/WebAssembly/wasi-nn) *like* style contract to make inference engine requests
+10. use a model path to fetch model data from a bindle server
+11. interact with some form of inference engine via Rust crate
 
-![legend for most basic design](images/legend_more_advanced_design.png "Most basic design of an example application doing inference")
+### Bindle Server
 
-</div>
+The [bindle server](https://github.com/deislabs/bindle) shall
 
-#### Data flow
+12. Be deployed "out of the box" as is as a standalone entity for storing/exposing models
+13. reference the respective model via a __model path__
```

Comment:
Same comment re: numbering

Reply:
Sorry, same words as above, catched the wrong cell:

This is supposed a feature, not a bug. My intention is to assign each of these requirements a unique id such that we may easily reference them, for example, when we decide to change it would be easy to indicate that in some history notes. However, if you like to have that changed I do not mind.